lisa/sut_orchestrator/azure/common.py

```diff
@@ -186,6 +186,7 @@ def __hash__(self) -> int:
 class VhdSchema:
     vhd_path: str = ""
     vmgs_path: Optional[str] = None
+    is_accelerated_network_supported: Optional[bool] = None
```

Comment:
Add a base class for marketplace, sig, and vhd. So this field can be used on all of them.

Reply:
From below logic, it doesn't need to be optional. So, set default value to `True`.

Reply:
> Add a base class for marketplace, sig, and vhd. So this field can be used on all of them.

For marketplace image and SIG, users don't need to set this field. The AN support information can be got from image_info.

Reply:
There are some cases, the image info is not right, or lack of image info. So for SIG and marketplace images, it loads image info firstly. If there are user set information, then override image info.

Reply:
Have added.